Failing mixer module tests

When running the unit tests locally on windows (Python 3.9 64 bit)  I get three failures:

```
.........s......F...FF..u.............
======================================================================
FAIL: test_init__keyword_args (__main__.MixerModuleTest)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "C:\Users\dan\Programming\pygame\test\mixer_test.py", line 66, in test_init__keyword_args
    self.assertEqual(mixer_conf[2], CONFIG["channels"])
AssertionError: 6 != 2

======================================================================
FAIL: test_pre_init__keyword_args (__main__.MixerModuleTest)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "C:\Users\dan\Programming\pygame\test\mixer_test.py", line 78, in test_pre_init__keyword_args
    self.assertEqual(mixer_conf[2], CONFIG["channels"])
AssertionError: 6 != 2

======================================================================
FAIL: test_pre_init__zero_values (__main__.MixerModuleTest)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "C:\Users\dan\Programming\pygame\test\mixer_test.py", line 86, in test_pre_init__zero_values
    self.assertEqual(mixer.get_init(), (44100, -16, 2))
AssertionError: Tuples differ: (44100, -16, 6) != (44100, -16, 2)

First differing element 2:
6
2

- (44100, -16, 6)
?              ^

+ (44100, -16, 2)
?              ^


----------------------------------------------------------------------
Ran 38 tests in 3.374s

FAILED (failures=3, skipped=1, unexpected successes=1)
```

Looks like setting the number of channels to 2 isn't working?

--- a/test/mixer_test.py
+++ b/test/mixer_test.py
@@ -63,7 +63,7 @@ class MixerModuleTest(unittest.TestCase):
         self.assertEqual(mixer_conf[0], CONFIG["frequency"])
         # Not all "sizes" are supported on all systems,  hence "abs".
         self.assertEqual(abs(mixer_conf[1]), abs(CONFIG["size"]))
-        self.assertEqual(mixer_conf[2], CONFIG["channels"])
+        self.assertGreaterEqual(mixer_conf[2], CONFIG["channels"])
 
     def test_pre_init__keyword_args(self):
         # note: this test used to loop over all CONFIGS, but it's very slow..
@@ -75,7 +75,7 @@ class MixerModuleTest(unittest.TestCase):
         self.assertEqual(mixer_conf[0], CONFIG["frequency"])
         # Not all "sizes" are supported on all systems,  hence "abs".
         self.assertEqual(abs(mixer_conf[1]), abs(CONFIG["size"]))
-        self.assertEqual(mixer_conf[2], CONFIG["channels"])
+        self.assertGreaterEqual(mixer_conf[2], CONFIG["channels"])
 
     def test_pre_init__zero_values(self):
         # Ensure that argument values of 0 are replaced with
@@ -83,7 +83,9 @@ class MixerModuleTest(unittest.TestCase):
         mixer.pre_init(22050, -8, 1)  # Non default values
         mixer.pre_init(0, 0, 0)  # Should reset to default values
         mixer.init()
-        self.assertEqual(mixer.get_init(), (44100, -16, 2))
+        self.assertEqual(mixer.get_init()[0], 44100)
+        self.assertEqual(mixer.get_init()[1], -16)
+        self.assertGreaterEqual(mixer.get_init()[2], 2)
 
     def test_init__zero_values(self):
         # Ensure that argument values of 0 are replaced with

